.github/CONTRIBUTING.md

```diff
@@ -62,17 +74,30 @@ To submit a pull request for Forseti, follow the process below:
          git checkout -b YOUR_DEVELOPMENT_BRANCH
 
   1. Create your change.
-    - Don't incorporate multiple changes in one PR. A change should be a logical, self-contained unit of work, feature, or fix. This simplifies troubleshooting and rollbacks.
-    - Learn how to execute [Inventory](inventory-quickstart), [Scanner](scanner-quickstart), or [Enforcer](enforcer-quickstart).
+    - Don't incorporate multiple changes in one PR. A change should be a
+    logical, self-contained unit of work, feature, or fix. This simplifies
+    troubleshooting and rollbacks.
+    - Learn how to execute
+    [Inventory](http://forsetisecurity.org/docs/quickstarts/inventory/)
+    [Scanner](http://forsetisecurity.org/docs/quickstarts/scanner/), or
+    [Enforcer](http://forsetisecurity.org/docs/quickstarts/enforcer/).
   1. Create tests for your change.
-    - You should write applicable unit tests for your changes, especially for changes involving substantial logic.
-    - Learn how to [run the tests](testing).
+    - You should write applicable unit tests for your changes, especially for
+    changes involving substantial logic.
+    - Learn how to
+    [run the tests](http://forsetisecurity.org/docs/development/#executing-tests).
   1. Commit your changes and push them to your development branch:
 
          git push origin YOUR_DEVELOPMENT_BRANCH
 
-  1. Open a [pull request](https://help.github.com/articles/about-pull-requests/) to begin the code review.
-    - All submissions, including submissions by project members, require a code review.
-    - The Github UI will display whether you have any merge conflicts that need to be resolved.
-    - All tests must pass before we review your PR. To test locally, you can install the git hooks under `scripts/githooks`.
-  1. After your PR is approved and all merge conflicts are resolved, we'll merge your PR.
+  1. Open a
```

Comment:
The numbering looks broken-ish:
screenshot
![image](https://user-images.githubusercontent.com/6977544/27556088-0cd67578-5a69-11e7-8322-6ef9397dba62.png)